test/functional/feature_bip8.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+# Test BIP 8 softforks
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+class Bip8Test(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 3
+        self.setup_clean_chain = True
+        # Node 0 has TestDummy inactive
+        # Node 1 has a regular activation window
+        # Node 2 uses speedy trial.
+        self.extra_args = [
+            ['-vbparams=testdummy:@1:@1'],
```

Comment:
Note `@1:@1` gets interpreted equivalently to `@144:@144` and transitions from DEFINED to FAILED for the block at height 144.

Reply:
Since this is for a node that has testdummy inactive, I've just changed this to `@-2:@-2`.

Reply:
I don't think that works? "never active" means the fork won't show up in getblockchaininfo (to avoid testdummy appearing on mainnet).

Reply:
I work around that by allowing a status of `None` to indicate that it shouldn't show up in `getblockchaininfo`. The purpose of this node is to generate non-signaling blocks anyways, so the status of testdummy for it doesn't matter that much.